stat/roc.go

```diff
@@ -98,15 +98,19 @@ func ROC(cutoffs, y []float64, classes []bool, weights []float64) (tpr, fpr, thr
 		}
 		nPos += posWeight
 		nNeg += negWeight
-		if y[i] <= cutoffs[bin] {
+		// Count false negatives (in tpr) and true negatives (in fpr).
+		if y[i] < cutoffs[bin] {
 			tpr[bin] += posWeight
 			fpr[bin] += negWeight
 		}
 	}
+	// Bins beyond the maximum value in y are skipped
+	// leaving these fpr and tpr elements as zero.
```

Comment:
Sorry, this should be after L112, just before the for loop.

Reply:
Oh yeah that makes sense